build-tools/flickrdownloadr_launch_linux.sh

```diff
@@ -47,12 +47,12 @@ FD_FIRST_RUN="${XDG_CONFIG_HOME:-$HOME/.config}/flickr-downloadr/firstrun"
 if [ ! -f "$FD_FIRST_RUN" ]; then
   mkdir -p `dirname "$FD_FIRST_RUN"`
   echo $(date +"%Y-%m-%d_%H-%M-%S") > ${FD_FIRST_RUN}
-  mozroots --import --sync
+  sudo cert-sync /etc/ssl/certs/ca-certificates.crt
 fi
 
 # this script should be in $PREFIX/bin
 MONO_EXEC="exec -a flickrdownloadr mono-sgen"
-EXE_PATH="${0%%/flickrdownloadr}/bin/flickr-downloadr.exe"
+EXE_PATH="/opt/flickrdownloadr/bin/flickr-downloadr.exe"
```

Comment:
IIRC, the path was not being set to `/opt/flickrdownloadr` when you choose defaults and install using the downloadable installer (`run.sh`, which unfortunately has been tested only on Ubuntu) from [here](https://flickrdownloadr.com/downloads/) - [32-bit](https://flickrdownloadr.com/installer/linux/flickrdownloadr-2.0.0.1-linux-installer.run) or [64-bit](https://flickrdownloadr.com/installer/linux-x64/flickrdownloadr-2.0.0.1-linux-x64-installer.run)

Could you kindly check if your fixes for making the `*.deb` installation would still work if we just leave `EXE_PATH` to be as it is?